src/settings/JobProfiles/resultsFormatter.js

```diff
@@ -36,11 +43,20 @@ export const resultsFormatter = {
   tags: record => {
     const tags = get(record, 'tags.tagList', []);
 
-    if (!tags.length) {
+    if (isEmpty(tags)) {
       return '-';
     }
 
-    return tags.join(', ');
+    return (
+      <span className={css.tags}>
```

Comment:
Looks weird for me to use `span` with `display: flex`. Consider using `div` or `display: inline-flex` if it will not break the markup

Reply:
Agree on using `display: inline-flex`